### PR TITLE
Minor fixes and improvements for CNSMap

### PR DESCRIPTION
**Pull request by MMerzinger**
Fixed typos and other minor things. Removed content in wrong place.

### CODE REVIEW

File: content/security-policies.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ date: "2020-10-01"
...
+It is essential to consider the security features and various configuration options of your orchestrator to control the security privileges the container runtime can use to spawn containers. The use of higher level policy and governance constructs may enforce those security guardrails.
 
 
 ## Projects
 - [OPA](https://www.openpolicyagent.org/)
 - [Kubernetes Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/)
 - [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
-- [Kubernetes Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) 
+- [Kubernetes Pod Security Policy (PSP) - (deprecated, removed in 1.25)](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) 
+- [Kubernetes PSP Replacement Policy (alpha in 1.22)](https://github.com/kubernetes/enhancements/issues/2579)
```
Comment (ashutosh-narkar) [inline content/security-policies.md:15]: Nit: We probably don't need to include this right now. Once there is an implementation we can add this in. WDYT ?

**Review by ashutosh-narkar — COMMENTED** [review]
Thanks for this contribution @MMerzinger! Just one minor comment, rest lgtm.

File: content/security-policies.md (hunk 1 above)
Comment (MMerzinger) [reply content/security-policies.md]: I agree that it is too early for PSP replacement. Perhaps the reference to the deprecation of PSP would be useful, as the target group of readers may not know the stage of the lifecycle. But I have removed it with a new commit as it is clearly visible when you go to the link for PSP.

**Comment by MMerzinger** [comment]
FYI: I added the sigstore and Connaisseur projects under Image Trust & Content Protection.

**Comment by MMerzinger** [comment]
@ashutosh-narkar I fixed the merge conflicts and added myself to the contributors. Is the PR good for you or is there anything else to fix/change?

File: content/industries.md [hunk 2]
```diff
@@ -40,19 +40,3 @@ public interest. In the public sector, it is essential to maintain harmony and t
...
-- [Linkerd](https://github.com/linkerd)
-- [HyperLedger](https://www.hyperledger.org/)
-- [HashiCorp Vault](https://github.com/hashicorp/vault)
-- [Keylime](https://keylime.dev/)
-- [Kyverno](https://github.com/kyverno/kyverno)
-- [Keycloak](https://github.com/keycloak/keycloak)
-
-## Examples
-- Adopt a Zero trust architecture with least-privilege security policies. See [this](zero-trust-architecture#examples)
-  for more examples.
```
Comment (ashutosh-narkar) [inline content/industries.md:58]: Just for my understanding, why did you remove this ?

Comment (MMerzinger) [reply content/industries.md:58]: From the perspective of the reader, each industry has its challenges and these challenges needs to be solved with e.g. a CNCF project. But currently there is no link between an industry and a project, hence it was confusing finding a list of projects that is almost a one to one copy from zero-trust-architectures. So I believe there is not "just" zero trust, that is important for those industries but even more topics and these are all covered by other chapters.

Comment (ashutosh-narkar) [reply content/industries.md:58]: Thanks for the clarification.

**Comment by ashutosh-narkar** [comment]
@MMerzinger can you please squash your commits and then we can merge this.

**Comment by MMerzinger** [comment]
@ashutosh-narkar Okay like this or shall I squash the last two commits as well?

**Review by ashutosh-narkar — APPROVED** [review]
